### PR TITLE
feature/setup

### PR DESCRIPTION
**Pull request by ludmilka-k**

### CODE REVIEW

**Comment by dkBrazz** [comment]
cool

File: src/css/styles.css [hunk 1]
```diff
@@ -12,13 +12,72 @@
   font-size: 16px;
   line-height: 24px;
   font-weight: 400;
-
-  color: #242424;
-  background-color: rgba(255, 255, 255, 0.87);
+  
+  --main-font-family: 'Inter', sans-serif;
+  --text-color: #242424;
+  --background-color: rgba(255, 255, 255, 0.87);
```
Comment (dkBrazz) [inline src/css/styles.css:18]: ,bzxmnbsmnxz

File: src/css/styles.css [hunk 2]
```diff
@@ -12,13 +12,72 @@
...
+body {
+  font-family: var(--main-font-family);
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: calc(24/16);
+  letter-spacing: 0.02em;
+
+  color: var(--text-color);
+}
```
Comment (dkBrazz) [inline src/css/styles.css:37]: mdsbvmsgdbcmsabdmcasbm

File: src/css/styles.css [hunk 3]
```diff
@@ -12,13 +12,72 @@
...
+.visually-hidden {
+    position: absolute;
+    white-space: nowrap;
+    width: 1px;
+    height: 1px;
+    margin: -1px;
+    border: 0;
+    padding: 0;
+    clip: rect(0 0 0 0);
+    clip-path: inset(50%);
```
Comment (dkBrazz) [inline src/css/styles.css:81]:
```suggestion
    clip-path: inset(70%);
```
mnbzdmncbasmncbasmcbasmn

**Review by dkBrazz — CHANGES_REQUESTED** [review]
mgajmhscgsdmhzgcmas

File: src/css/styles.css (hunk 1 above)
Comment (dkBrazz) [reply src/css/styles.css:18]: ,jsdghvmsdgb

File: src/css/styles.css (hunk 2 above)
Comment (dkBrazz) [reply src/css/styles.css:37]: asmncbasmncbvamscn

File: src/css/styles.css (hunk 1 above)
Comment (ludmilka-k) [reply src/css/styles.css:18]: dfbdfbdfbv

File: src/css/styles.css (hunk 2 above)
Comment (ludmilka-k) [reply src/css/styles.css:37]: rgfrg

**Review by dkBrazz — APPROVED** [review]

**Pull request merged**